### PR TITLE
Add reminder which site uses which Bouncer/SA service

### PR DESCRIPTION
**Pull request by stevejalim**

### CODE REVIEW

**Review by alexgibson — APPROVED** [review]
r+wc

File: docs/pipeline.rst [hunk 1]
```diff
@@ -29,24 +29,31 @@ Dev
 - *Pocket Marketing pages URL:* https://dev.tekcopteg.com/
 - *Bedrock locales:* dev repo
 - *Bedrock Git branch:* main, deployed on git push
+- *Firefox download URLs:* Bouncer staging
```
Comment (stevejalim) [inline docs/pipeline.rst:32]:
```suggestion
- *Firefox download URL:* https://bouncer-bouncer.stage.mozaws.net/
```

File: docs/pipeline.rst [hunk 2]
```diff
@@ -29,24 +29,31 @@ Dev
...
 
 Staging
 ~~~~~~~
 - *Mozorg URL:* https://www.allizom.org/
 - *Pocket Marketing pages URL:* https://www.tekcopteg.com/
 - *Bedrock locales:* prod repo
 - *Bedrock Git branch:* stage, deployed on git push
+- `Stub Attribution`_ service: Bouncer production
```
Comment (stevejalim) [inline docs/pipeline.rst:40]:
```suggestion
- *Firefox download URL:* https://download.mozilla.org/
```

File: docs/pipeline.rst [hunk 3]
```diff
@@ -29,24 +29,31 @@ Dev
...
 
 Production
 ~~~~~~~~~~
 - *Mozorg URL:* https://www.mozilla.org/
 - *Pocket Marketing pages URL:* https://getpocket.com/
 - *Bedrock locales:* prod repo
 - *Bedrock Git branch:* prod, deployed on git push with date-tag
+- `Stub Attribution`_ service: Bouncer production
```
Comment (stevejalim) [inline docs/pipeline.rst:48]:
```suggestion
- *Firefox download URL:* https://download.mozilla.org/
```

File: docs/pipeline.rst [hunk 4]
```diff
@@ -228,3 +235,5 @@ may be commits on that branch which aren't in yours – so, if you have the
 .. _public staging environment: https://www.allizom.org
 .. _Docker Hub: https://hub.docker.com/r/mozmeao/bedrock/tags
 .. _production: https://www.mozilla.org
+.. _demo sites: https://bedrock.readthedocs.io/en/latest/contribute.html#demo-sites
+.. _Stub Attribution: https://bedrock.readthedocs.io/en/latest/attribution/0002-firefox-desktop.html
```
Comment (stevejalim) [inline docs/pipeline.rst:239]:
```suggestion
```

File: docs/pipeline.rst [hunk 5]
```diff
@@ -29,24 +29,31 @@ Dev
...
 ~~~~~~~~~~
 - *Mozorg URL:* https://www.mozilla.org/
 - *Pocket Marketing pages URL:* https://getpocket.com/
 - *Bedrock locales:* prod repo
 - *Bedrock Git branch:* prod, deployed on git push with date-tag
+- `Stub Attribution`_ service: Bouncer production
 
-You can check the currently deployed git commit by checking /revision.txt on any of these URLs.
+.. note::
+    By default, the Demo servers on GCP point to the Bouncer Dev service.
```
Comment (stevejalim) [inline docs/pipeline.rst:51]:
```suggestion
    By default, the Demo servers on GCP point to the Bouncer Dev service at https://dev.bouncer.nonprod.webservices.mozgcp.net/
```

**Pull request merged**